docs/cmd/completion.md

```diff
@@ -43,10 +43,10 @@ Start a new shell for this setup to take effect.
 
 #### Fish completions
 ```bash
-containerlab completion fish | source
+❯ containerlab completion fish | source
 ```
 
 To load completions for each session, execute once:
 ```
-containerlab completion fish > ~/.config/fish/completions/containerlab.fish
+❯ containerlab completion fish > ~/.config/fish/completions/containerlab.fish
```

Comment:
the rule I started to follow is do not use shell sign if the command doesn't have output displayed. This is in accordance to some linter rules, as well as it makes it possible to copy the command using the copy icon and not using your mouse to copy the portion of the command without the shell sign

so please revert this change